subprojects/packagefiles/lmdb/meson.build

```diff
@@ -0,0 +1,42 @@
+project('lmdb', 'c', version: '0.9.29', license: 'OpenLDAP')
+
+cc = meson.get_compiler('c')
+
+src = [
+  'libraries/liblmdb/mdb.c',
+  'libraries/liblmdb/midl.c'
+]
+
+docs = [
+  'libraries/liblmdb/mdb_stat.1',
+  'libraries/liblmdb/mdb_copy.1',
+  'libraries/liblmdb/mdb_dump.1',
+  'libraries/liblmdb/mdb_load.1'
+]
+
+c_args = [
+  '-Wno-unused-parameter',
+  '-Wbad-function-cast',
+  '-Wuninitialized'
+]
+
+threads_dep = dependency('threads', required: true)
+incdir = include_directories('libraries/liblmdb')
+
+lmdb = library(
+  'lmdb',
+  src,
+  c_args: c_args,
+  dependencies: threads_dep,
+)
+
+lmdb_dep = declare_dependency(include_directories: incdir, link_with: lmdb)
+install_man(docs)
+
+if get_option('programs')
+  subdir('programs')
+endif
```

Comment:
If you are going to make an option to install the programs, then the man pages for those programs should only be installed when the programs are.